src/tests/ftest/vmd/fault_reintegration.py

```diff
@@ -166,16 +169,11 @@ def test_nvme_fault_reintegration(self):
         self.log_step(
             "Marking the {} device as faulty and verifying it is 'EVICTED' and its "
             "LED is 'ON'".format(test_dev))
-        check = self.check_result(
-            set_device_faulty(self, self.dmg, self.dmg.hostlist, test_dev, self.pool),
-            "EVICTED", "ON")
-
-        if not check:
-            self.fail("#Result of set_device_fault, device not in expected EVICTED, ON state")
+        set_device_faulty(self, self.dmg, self.dmg.hostlist, test_dev, self.pool)
```

Comment:
Why don't we need `check_result` anymore?

Reply:
@daltonbohning  : During the testing we noticed the set_device_faulty output which is incompatible with the check_result method.